src/libraries/System.IO.Pipes/ref/System.IO.Pipes.csproj

```diff
@@ -1,12 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);net5.0;</TargetFrameworks>
```

Comment:
You need to remove this. Pipes is part of the shared framework, not a package.  We shouldn't rebuild for old TFMs for things that aren't packages, since those builds would never ship.

Reply:
We could add an error to our build system to catch this problem.